src/index.scss

```diff
@@ -463,6 +463,12 @@ header {
   margin-bottom: 0px;
 }
 
+code {
+  word-wrap: break-word !important;
+  text-wrap: initial;
```

Comment:
please you !important here as well, so that I does not get overridden by something else in future